.rubocop.yml

```diff
@@ -0,0 +1,27 @@
+AllCops:
+  TargetRubyVersion: 2.3
+  Exclude:
+    - 'modules/**/*'
+    - 'vendor/**/*'
+Layout/AlignHash:
+  EnforcedHashRocketStyle: table
+Style/StringLiterals:
```

Comment:
GPLT uses the following based on our discussion of single vs. double quotes:
```
Style/StringLiterals:
  EnforcedStyle: double_quotes
```

Should we enforce double quotes here as well for consistency?

Reply:
+1   yes please.

Reply:
The enforcement of StringLiterals and FrozenStringLiteralComment were disabled to prevent a massive change set as part of this PR.  This work has been ticketed in https://tickets.puppetlabs.com/browse/SLV-678 to update these configuration settings and fix the code accordingly.